backend/src/main/java/corea/room/controller/RoomController.java

```diff
@@ -1,31 +1,38 @@
 package corea.room.controller;
 
+import corea.auth.annotation.LoginMember;
+import corea.auth.domain.AuthInfo;
+import corea.member.domain.Member;
 import corea.room.dto.RoomResponse;
 import corea.room.dto.RoomResponses;
 import corea.room.service.RoomService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/rooms")
 @RequiredArgsConstructor
 public class RoomController {
```

Comment:
```suggestion
@RequestMapping("/rooms")
public class RoomController {
```
RequestMapping 어떤지 🙂🙂

Reply:
깃 충돌이 불러일으킨 폐해랄까.. 아무튼 바꿔용~